strings/memories.json

```diff
@@ -411,5 +411,14 @@
 	"supermatter_dusted_moods":[
 		"%PROTAGONIST %MOOD as they faded way.",
 		"%PROTAGONIST %MOOD as they are reduced to atoms."
+	],
+	"nuke_code_names":[
+		"The nuke code for the %PROTAGONIST, %NUKE_CODE" 
+	],
+	"nuke_code_starts":[
+		"%PROTAGONIST starts grinning recalling the nuke code, %NUKE_CODE"
```

Comment:
```suggestion
		"%PROTAGONIST starts grinning as they recall the code for detonating a nuclear device, %NUKE_CODE"
```

Reply:
Sure, I can do that, unfortunately when I was writing this it was at like 5 am and I literally couldn't think of what to write at all, these are better tbh